src/main/java/golf/servletquest/repository/StepRepository.java

```diff
@@ -0,0 +1,41 @@
+package golf.servletquest.repository;
+
+import golf.servletquest.entity.Step;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class StepRepository {
```

Comment:
Please base your logic on abstraction / not implementation. We need to create an interface for current implementation